CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+-   Fix on tooltip that would persist even after the mouse has been moved out of the anchor.
+
+### Changed
+
+-   _[BREAKING]_ `Popover.useComputePosition` hook has been removed.
+-   _[BREAKING]_ `Offset` type does not have `vertical` and `horizontal` keys anymore but has `along` and `away` props.
+-   _[BREAKING]_ `Popover` computes position internally and add popover ref internally. Click away and escape are handled inside the popover. The popover can now have an anchor. `popoverRect`, `isVisible` and `popoverRef` props have been removed. `anchorRef`, `placement`, `offset`, `fitToAnchorWidth`, `isOpen`, `closeOnClickAway`, `closeOnEscape`, `onClose` and `hasArrow` props have been added.
```

Comment:
I think I would move the `hasArrow` into a separated change like this:

```md
### Added
-      `Popover` now can show an arrow pointing on the anchor using the `hasArrow` prop.
```

Reply:
And actually `closeOnClickAway` and `closeOnEscape` also count as new features for the popover. It's nice to be a little positive and show we don't just break stuff but also add features :D